cmd/starter/c/starter.c

```diff
@@ -265,6 +265,32 @@ static void prepare_scontainer_stage(int stage) {
         }
     }
 #endif
+
+    if ( config.isMinimalDev ) {
+        if ( isatty(STDIN_FILENO) || isatty(STDOUT_FILENO) || isatty(STDERR_FILENO) ) {
```

Comment:
@DrDaveD Could you move this code into `internal/pkg/runtime/engines/singularity/process.go` rather in order to make it specific to singularity engine because that could break other engines (eg: OCI engine). Like that there is no need for `isMinimalDev` and associated getter/setter methods for starter config